getting-started/get-the-sdk.md

```diff
@@ -2,7 +2,7 @@
 
 The Adobe Experience Platform SDK is available for iOS via [Cocoapods](https://cocoapods.org/), Android via [Gradle](https://gradle.org), and for React Native projects via [Node Package Manager](https://nodejs.org).
 
-Follow the directions below to incorporate the SDK into your mobile application.
+Follow the directions below to to incorporate the SDK into your mobile application.
```

Comment:
I'm guessing this is a spillover from the previous PR...there are 2 "to"s again!